OpenRA.Mods.Common/UpdateRules/Rules/20221203/ExplicitSequenceFilenames.cs

```diff
@@ -330,6 +336,10 @@ public override IEnumerable<string> UpdateSequenceNode(ModData modData, MiniYaml
 
 		void ProcessNode(ModData modData, MiniYamlNode sequenceNode, MiniYamlNode resolvedSequenceNode, string imageName)
 		{
+			// "Filename" was introduced with this update rule, so that means this node was already processed and can be skipped
+			if (sequenceNode.LastChildMatching("Filename") != null)
+				return;
```

Comment:
Are we sure this won't break the rule for updating mods? This doesn't queue changes like #20861 does

Reply:
I don't think we are processing the same node several times? The update rule doesn't behave properly when this node already exist anyway.

Reply:
I meant in regards to reading defaults node. And it also seems this is completely unnecessary change. Update rules are not meant to be run on the same yaml twice. It's a waste of effort to try to make that a possibility.

Reply:
They are not meant to but might be. That is how this bug was detected after all. This PR would have been avoided if not for this very dismissive attitude.

Reply:
In quite some cases, it's impossible or extremely hard to make rules rerun-proof. And since we won't make sure all are safe, i don't think it makes any sense to put in effort into any making any of them rerun-safe.

Reply:
The change has already been made. You are arguing to put more effort into this. I'm not saying we should go out of our way to build super complicated update rules, but the change is very easy here and good practice.

Reply:
There is still effort left to put. We need to test whether this doesn't break mod updating

Reply:
a) Impossible, see the comment.
b) I already did regardless, and the automatic rules produce the same result as #20580.

Reply:
ok I'm going to trust that